pkg/controller/atlas/client.go

```diff
@@ -0,0 +1,21 @@
+package atlas
+
+import (
+	"github.com/mongodb-forks/digest"
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/util/httputil"
+	"go.mongodb.org/atlas/mongodbatlas"
+	"go.uber.org/zap"
+)
+
+// AtlasClient is the central place to create a client for Atlas using specified API keys and a server URL.
+// Note, that the default HTTP transport is reused globally by Go so all caching, keep-alive etc will be in action.
+func AtlasClient(connection Connection, log *zap.SugaredLogger) (*mongodbatlas.Client, error) {
```

Comment:
[nit] given the name of the package is `atlas`, `atlas.AtlasClient` is a bit redundant, please consider something like `atlas.NewClient`

Reply:
Yes!